client/src/pages/Organization/create-event/index.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import {
+  Button, Typography, Modal, Box,
+} from '@mui/material';
+
+import MyStepper from './stepper';
+
+import './style .css';
+
+const EventStepper = () => {
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
```

Comment:
you can handle `open & close` in one function 
```suggestion
  const handleOpen = () => setOpen(!open);
```